commands/release/release.go

```diff
@@ -43,9 +44,10 @@ const (
 	extraNanosecondsToAddToLastFetchedTimestamp = 0
 	lastFetchedFileMode                         = 0644
 
-	relChangelogFilepath                      = "docs/changelog.md"
-	relAPIFolderPathForKurtosisMonorepo       = "api/"
-	typeScriptPackageJSONForMinimalGRPCServer = "typescript/package.json"
+	relChangelogFilepath = "docs/changelog.md"
+
+	gitIgnoreFileRelPath      = ".gitignore"
```

Comment:
Breaks the {abs,rel}{dir,file}path naming convention (I'd personally call this `gitIgnoreRelFilepath` and add a comment that it's relative to repo root